merkle/crdt/merklecrdt.go

```diff
@@ -11,15 +11,16 @@
 /*
 Package crdt provides CRDT implementations leveraging MerkleClock.
 */
-package crdt
+package merklecrdt
```

Comment:
suggestion: as far as I know the best practice in Go is to have package name match the folder it in.

I know that we have another `crdt` package and you renamed it here to make them more distinct. But maybe it makes sense to rename the folder as well `merkle/merklecrdt`?

Another great benefit of it is that it will automatically resolve imports.

Reply:
It's best practice but often ignored when the folder name is not ideal. Like in cases where the folder name is `graphql-go` for example. and we just want to use `graphql`. As for `merkle/merklecrdt`, I much rather keep it as `merkle/crdt`.